content/automations/_index.en.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Automations and Bots"
+date: 2025-02-17T19:11:00+02:00
+chapter: false
+draft: false
+---
+
+If you want to automatically generate messages in a Matrix room, the API does not work directly due to SSO authentication. Instead, you can use the Academiccloud Bot instance for this purpose.
+
+There are many possible use cases, such as:
+
+- Automated welcome messages for new room members
+- Creating toots on Mastodon
+- Automatic reminders for scheduled events
+- Notifications for changes in software repositories
+- Automatic alerts for disruptions in web services
+- ...
+
+Depending on the use case, a Hookshot user can be added to a room with the necessary permissions. This user can then be used in a few simple steps for RSS feeds, GitLab events, or webhooks.
+
+If you need a more general solution to respond to certain room events (e.g., a user joining), a bot account can be created. This bot can send messages via the Client-Server API, and messages will appear in the room as coming from the bot.
+
+## Hookshot-User {#hookshot-user}
+To use hookshot, you must invite the hookshot user `@hookshot:gwdg.de` from the main instance into a room and grant it power level 'Moderator'. After that, you can subscribe to an RSS feed or create a new webhook with the following chat messages:
```

Comment:
oben ist hookshot immer groß geschrieben

Reply:
ach und mach mal das mit dem power level moderator fett oder als Stichpunkt in ner eigenen Zeile oder so